### DISCUSSION TITLE
Extend IDS RAM / Process Layer

### DISCUSSION THREAD

**Discussion by anilturkmayali** [International-Data-Spaces-Association/Personal-Data-and-IDS#11] [Ideas]
**What could be elaborated?** 
Processes and workflows that ensure the secure and compliant handling of personal data within the IDS ecosystem.

**Action Steps:**
- Map out data flow processes (data sharing patterns) that involve personal data.
- Review existing methods for data anonymization and pseudonymization within IDS processes.
- Propose process optimizations that prioritize user-centricity and data sovereignty.

**Comment by mabuse68** [comment]
Let’s tackle consent.

It would be nice to have a way to manage personal information consent as a standardised (semantically), and integrated negotiation, preceding the effective data sharing.

I happen to see a “hot spot” in the notion of “Intended Usage”. It helps its application to a wide range of use cases:

I took inspiration from this project: https://github.com/EBISPOT/DUO
Where the data is discoverable if it matches the intended usage; the intended usage has been described in informed consent ontology (it’s healthcare).

To elaborate my suggestion: we could research an IDS “intended use taxonomy”, or perhaps adopt the existing ones. We could then describe ODRL policy and operators that match(es) the intended use.
I would focus on this topic first, because creating the framework to manage “intended use”, solves a lot of use cases where consent is a requirement.

For instance: Data sharing negotiation (IDS)
If the data asset “carries” collective consent with it, it’s only a matter of checking if the usage complies with the scope of consent. 
We are back to the previous case (As I said, these use case can complement one with each other).

If consent is available at individual level (see, a SOLID ecosystem), we need to scale consent matching in an economical way, so, if each data owner agrees on an “intended use”, it makes things easier.

And during Data request phase (IDS)
When data is requested, we create a historical trace of consent/intended usage tuples. Backtracking by intended usage is a good way to determine if data with a complex provenance can be re-used for a specific use case. Data intermediaries could play a role in keeping a registry of these usage transactions.

Informed consent ontology for medical procedures
https://www.ebi.ac.uk/ols4/ontologies/ico
OVerview of consent ontologies
https://www.researchgate.net/publication/352211926_Consent_Through_the_Lens_of_Semantics_State_of_the_Art_Survey_and_Best_Practices 

WDYT?

**Reply by AlexTourski** [reply]
Dear Gabriele, may I suggest that we consider the use case first? Could you please present it? The problem is that if you put it in the context of the current IDS, it sill sound like "As a patient, I need to provide consent specifically via my hospital, my doctor, my insurance company, my ministry of health, Apple and 23 other organizations which keep my medical data. And I will have to explain to all of them which groups of people (in their respective context they have different IDs and groups) can access my data". 
In the context of Solid it will be "As a patient I provide consent for certain groups to access my data on my POD on my conditions". 
You can imagine that it will be two completely different stories, with two different protocols to support them, right?

**Reply by FelixBole** [reply]
I'd like to introduce the concept of the Personal Data Intermediary (PDI) in this discussion. It seems to align well with the ideas. PDIs are designed to stramline consent management by allowing individuals to easily grant, manage and revoke consent for data usage. This process is underpinned by a clear understanding of how, why and by whom the data is used, which aligns with the notion of Intended Usage that you are mentionning.

Also, it's worth noting that the PDI's consent management system is envisioned to support a wide range of use cases, including healthcare, by leveraging informed consent ontologies and potentially adopting an 'Intended Use Taxonomy'. The integration of these concepts within the PDI framework could indeed facilitate more efficient data sharing negotiations and consent matching in ecosystems like IDS.

As for subjects regarding negotiation and policy management, I'd like to point out the ongoing work by [Prometheus-X](https://github.com/Prometheus-X-association) to essentially build components that enable the generation of data sharing agreement contracts, injection of ODRL policies and verification/validation of these policies. The aim of this work being to facilitate the creation, negotiation and enforcement of data sharing agreements based on ODRL standards. This feature ensures that data sharing aligns with the specified intended use and consent terms, thereby streamlining the process of data usage and sharing in compliance with the established guidelines. I mention these components along with the PDI because the PDI is said to communicate with such access and usage control services for the management and verification of agreement and policies.

**Comment by anilturkmayali** [comment]
Great & inspiring discussion, thank you. I've just added this into the agenda for the meeting on Friday (December 15), by allocating 25 mins. as we need really some time to discuss this. Based on the results of the discussion I'll try to turn it into a task for the group. 

Thank you!

**Reply by mabuse68** [reply]
I elaborate further on the problems we might want to solve during the lifecycle of a data sharing agreement.
It could be the waterline for the discussion of today:

Consent must be:
**Freely given**. Users have the right to consent or not based on the provided information. One should not be pressured to consent.  
**Specific**. Consent should be requested about specific data.  
Informed. Users are presented with information about the data controller’s identity.  
**Unambiguous**. Information should be provided in a “clear and plain” language.  
**Withdrawable** (this term might not even exist). Users must be aware of their right to revoke consent. Further, the revocation option should be clearly stated and easily accessible. Revoking consent must be as easy as granting it from an end-user perspective, specifically w.r.t. the data to be processed, how it is to be used and the purpose of the processing.  

However:

- We can’t burden a data sharing agreement with the infrastructure and traceability that are needed to implement these principles at individual level.
- We need to find the solution that enables these four principles, yet it requires minimum energy and sunken costs for the participants.
- If we look at this from the standpoint of the data consumer/producer:
- Individual consent is effectively an access barrier for those data initiatives that have less chances to appeal to a person. A social network can offer immediate gratification, much more than a life-saving medical research where personal data is needed. 
- Withdrawn consent should propagate across the whole data usage chain.
- A data asset has variable data quality (data coverage, accuracy, etc).
- Change of intended usage can be prevented, but not avoided at all, at that point a data producer could be confronted with the re-publishing of assets.

Perhaps Data intermediaries can play the role in making consent a **collective item** that can stick to a data product. In place of having to manage everyone's consent, let's "subscribe" everyone's to its intended usage.
A data intermediary could:
Identify the intended usage
Encode it in a recognisable, re-usable form
Publish it as a consent item
Attach mechanisms to link individuals to the intended usage object
Implement change management (for instance: how to communicate intended usage changes or new data consumers to the subscribed consent-givers)
Attach a visual representation to intended usage, people can understand that the green icon is better than the yellow one, and the red one is indulging in a dangerous life!

**Comment by anilturkmayali** [comment]
Maybe to have this kind of framework for all industries to have a methodology (starting with a questionnaire) for human-centricity assessment for data spaces. 
I don't know how we can present the end result but maybe as an extension to the currently existing IDS docs. 
 
| Industry      | Category             | Intended Usage          | Types of Personal Data Shared                  | Data Sharing Stage         | ODRL Policy Mapping                                  |
|---------------|----------------------|-----------------------------|-----------------------------------------------|----------------------------|-----------------------------------------------------|
| Education     | Personalized Learning| Tailoring Education         | Student learning style, performance data     | II) Data Processing            | ...                           |
|               | Educational Research | Conducting Research         | Research survey data, collaboration records  | I) Data Sharing               | ...                         |
|               | Student Tracking     | Tracking Student Progress   | Student grades, attendance records            | III) Data Monitoring            | ...                         |
|               | E-Learning Platforms | Online Course Delivery      | Course progress, assessment results          | II) Data Sharing               | ...                         |
|               | Education Analytics  | Student Performance Analysis| Academic performance metrics, learning trends| IV) Data Analysis              | ...                        |

- target group should be added
